tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/RuntimeBrokerTests.cs

```diff
@@ -110,7 +110,7 @@ public async Task WamUsernamePasswordRequestAsync()
                .WithParentActivityOrWindow(windowHandleProvider)
                .WithAuthority(labResponse.Lab.Authority, "organizations")
                .WithLogging(testLogger)
-               .WithBrokerPreview().Build();
+               .WithWindowsBroker().Build();
```

Comment:
You should be able to add the same test for Desktop Broker as well

Reply:
There are pre-existing tests for WamBroker. I do not think any new are needed.